app/user/admin/admin.form.component.js

```diff
@@ -40,13 +41,14 @@
         // TODO: provide an endpoint that makes this one request instead of two.
         Api.all('admin')
```

Comment:
what's the purpose of getting the list of admins here? doesn't look like we're using that response data any more.